src/components/TextField/formatters/withCurrencyFormatting/test.tsx

```diff
@@ -249,4 +249,40 @@ describe('withCurrencyFormatting', () => {
 
         expect(component.find('input').prop('value')).toEqual('-19.12');
     });
+
+    it("should divide the value property by ten to the power of the maximum fraction digits, when the 'minor' property is true", () => {
```

Comment:
You don't really have to explain how the dividing works here IMO, maybe you improve the description by naming it something along the lines of:
```suggestion
    it('should use the value in minor units, when the "minor" prop is set', () => {
```
property can also be prop to avoid confusion with class properties.